www/gatsby-node.js

```diff
@@ -166,6 +166,11 @@ exports.createPages = ({ graphql, actions, reporter }) => {
     toPath: `/contributing/submit-to-creator-showcase/`,
     isPermanent: true,
   })
+  createRedirect({
+    fromPath: `/contributing/submit-to-creator-showcase/`,
+    toPath: `/showcase/`,
+    isPermanent: true,
+  })
```

Comment:
I'm not sure the best place to have the redirects go to

Reply:
This is correct